app/lib/swagger_docs.rb

```diff
@@ -54,6 +54,11 @@ class SwaggerDocs
     "<p>Note: for submissions after MTR Phase 2 is implemented, “disregarded payments” will be extended to cover “Modern Slavery Victim Care Contract (MSVCC) financial support payments”, and no longer cover “Back to Work Bonus payments”</p>",
   ].join
 
+  INACCESSIBLE_CAPITAL_TEXT = [
+    "Inaccessible capital - For submissions after MTR Phase 2 is implemented, capital where the value cannot be accessed to pay legal costs, should be excluded.",
```

Comment:
@starswan We need to wrap text in "p" tag

<img width="1181" alt="Screenshot 2023-09-28 at 11 01 44" src="https://github.com/ministryofjustice/cfe-civil/assets/195928/c4a59f9a-6c15-4ca4-a480-e86412fbf23c">